packages/components/src/InputNumber/InputNumber.tsx

```diff
@@ -1,20 +1,21 @@
 import React, { Ref, createRef, forwardRef, useImperativeHandle } from "react";
-import { FormField, FormFieldProps } from "../FormField";
+import { BaseFormFieldProps, FormField, FormFieldProps } from "../FormField";
 
-/**
- * The following is the same as:
- *   type BaseProps = Omit<FormFieldProps, "type" | "children">;
- * Unfortunately Docz doesn't currently support Omit so it has been reduced to
- * its component parts.
- */
 interface InputNumberProps
-  extends Pick<
-    FormFieldProps,
-    Exclude<
-      keyof FormFieldProps,
-      "type" | "children" | "rows" | "keyboard" | "actionsRef"
-    >
-  > {
+  extends BaseFormFieldProps,
+    Pick<
+      FormFieldProps,
```

Comment:
again, no need to extend both

Reply:
Discussed offline.  Changed `BaseFormFieldProps` to `CommonFormFieldProps` here: https://github.com/GetJobber/atlantis/pull/627/commits/f2c4161615bc5479b46b1aed9ca9f684700d1f90